<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewMusterDialog</class>
 <widget class="QDialog" name="NewMusterDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>317</width>
    <height>128</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="kinLabel">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Select Kindred:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="reaLabel">
     <property name="text">
      <string>Select Realm:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QComboBox" name="selectKindred">
     <item>
      <property name="text">
       <string>Albainn</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Anglecynn</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Atalantes</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Brythoniad</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Byzantii</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Erainn</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Formoraic</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Infernii</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Jutes</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Khthones</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Norse</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Vras</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Ysian</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QComboBox" name="selectRealm"/>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
     <property name="centerButtons">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>NewMusterDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>NewMusterDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
